kloppy/domain/models/common.py

```diff
@@ -303,6 +313,261 @@ def __eq__(self, other):
         return isinstance(other, Period) and other.id == self.id
 
 
+class Origin(Enum):
+    """
+    Attributes:
+        TOP_LEFT: Origin at the top left of the field
+        BOTTOM_RIGHT: Origin at the bottom left of the field
+        CENTER: Origin at the center of the field
+    """
+
+    TOP_LEFT = "top-left"
+    BOTTOM_LEFT = "bottom-left"
+    CENTER = "center"
+
+    def __str__(self):
+        return self.value
+
+
+@dataclass
+class CoordinateSystem(ABC):
+    normalized: bool
+    length: float = None
+    width: float = None
+
+    def __eq__(self, other):
+        if isinstance(other, CoordinateSystem):
+            return (
+                self.origin == other.origin
+                and self.vertical_orientation == other.vertical_orientation
+                and self.pitch_dimensions == other.pitch_dimensions
+            )
+
+        return False
+
+    @property
+    @abstractmethod
+    def provider(self) -> Provider:
+        raise NotImplementedError
+
+    @property
+    @abstractmethod
+    def origin(self) -> Origin:
+        raise NotImplementedError
+
+    @property
+    @abstractmethod
+    def vertical_orientation(self) -> VerticalOrientation:
+        raise NotImplementedError
+
+    @property
+    @abstractmethod
+    def pitch_dimensions(self) -> PitchDimensions:
+        raise NotImplementedError
+
+
+@dataclass
+class KloppyCoordinateSystem(CoordinateSystem):
+    @property
+    def provider(self) -> Provider:
+        return Provider.KLOPPY
+
+    @property
+    def origin(self) -> Origin:
+        return Origin.TOP_LEFT
+
+    @property
+    def vertical_orientation(self) -> VerticalOrientation:
+        return VerticalOrientation.TOP_TO_BOTTOM
```

Comment:
I'm not entirely sure we should default to `TOP_TO_BOTTOM`. I personally prefer the Cartesian coordinate system